test/integration/common.ts

```diff
@@ -15,6 +16,8 @@ import {
     NearClient,
     TerraClient,
 } from './clients';
+import { SolanaClient } from './clients/solana/clients';
+import { SolanaConfig } from './clients/solana/config';
```

Comment:
import from the index, just like the rest of the configs

Reply:
fixed